spec/features/verify_tokens_spec.rb

```diff
@@ -22,6 +22,7 @@
     }
     set_client = -> {
       client = BitPay::SDK::Client.new(api_uri: ROOT_ADDRESS, pem: PEM, insecure: true)
+      sleep 60 # rate limit compliance
```

Comment:
And we really, really shouldn't have to do this. Unless we're using test.bitpay.com for development work.